services/nuget-fixtures.js

```diff
@@ -34,9 +34,24 @@ const nuGetV3VersionJsonFirstCharNotZero = JSON.stringify({
   ],
 })
 
+const nuGetV3VersionJsonBuildMetadataWithDash = JSON.stringify({
+  data: [
+    {
+      totalDownloads: 0,
+      versions: [
+        {
+          version: '1.16.0+388',
+        },
+        { version: '1.17.0+1b81349-429' },
+      ],
```

Comment:
Interesting prettier, interesting 👀

Reply:
Prettier takes into account the formatting of the input file - its not only the AST structure that determines the output. I think with objects, prettier only switches you to/from single/multi-line objects if it also violates some other rule. If you change this to either:


```js
versions: [
  {
    version: '1.16.0+388'
  },
  {
    version: '1.17.0+1b81349-429'
  },
],
```

or

```js
versions: [
  { version: '1.16.0+388' },
  { version: '1.17.0+1b81349-429' },
],
```

and then run `prettier`, you won't get the result


```js
versions: [
  {
    version: '1.16.0+388',
  },
  { version: '1.17.0+1b81349-429' },
],
```

Reply:
That's my own fault for not checking it aftewards. I pasted it in expecting prettier to make them the same, but makes sense that it didn't (especially since it didn't exceed the max line length)

Reply:
Useful to know @chris48s ! @calebcartwright could you try fixing the formatting? 😄